core/src/main/java/org/elasticsearch/Version.java

```diff
@@ -313,6 +313,10 @@ public String toString() {
         return sb.toString();
     }
 
+    public String displayVersion() {
```

Comment:
@nik9000 `Version#toString` does not display whether or not the build is a snapshot build; I'm happy to add Javadocs?